pkg/apiserver/apiserver.go

```diff
@@ -113,8 +113,11 @@ const (
 
 // InstallREST registers the REST handlers (storage, watch, proxy and redirect) into a restful Container.
 // It is expected that the provided path root prefix will serve all operations. Root MUST NOT end
-// in a slash. A restful WebService is created for the group and version.
-func (g *APIGroupVersion) InstallREST(container *restful.Container) error {
+// in a slash.
+// If you are installing the API group to an existing webservice then it should be passed in.  If
```

Comment:
Hmm, I initially started out with a two method approach but switched just due to duplication.  Perhaps I can consolidate it nicely.  Will take another pass at that.